docs: add documentation to explain the extension demo (#179)
puppeteer/replay: README.md

  )
);
```

## Others

### Write code in CommonJS modules

If you want to use CommonJS modules, `require` the code from `@puppeteer/replay/cjs`. For example,

```js
const { stringify } = require('@puppeteer/replay/cjs');
```

### Create a Chrome extension for Recorder (Available from Chrome 104 onwards)

> You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extend DevTools.

Author: ```suggestion
You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extend DevTools.

This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to test the latest features in Chrome.
```

  )
);
```

## Others

> ### Write code in CommonJS modules

Reviewer: we don't need this (about CJS) anymore as it was outdated

> ### Write code in CommonJS modules
> 
> If you want to use CommonJS modules, `require` the code from `@puppeteer/replay/cjs`. For example,
> 
> ```js
> const { stringify } = require('@puppeteer/replay/cjs');
> ```
> 

Reviewer: ```suggestion
```

### Create a Chrome extension for Recorder (Available from Chrome 104 onwards)

You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extend DevTools.

> This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to test the latest features in Chrome.

Author: ```suggestion
This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to try out the latest bleeding edge features in Chrome.
```

> This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to try out cutting-edge features in Chrome.

Reviewer 2: ```suggestion
This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider installing [Chrome Canary](https://www.google.com/chrome/canary/) to try out cutting-edge features in Chrome.
```

> This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to try out the latest bleeding edge features in Chrome.

Author: ```suggestion
This feature only available from Chrome 104 onwards. Check your current Chrome version with `chrome://version`. Consider install [Chrome Canary](https://www.google.com/chrome/canary/) to try out cutting-edge features in Chrome.
```

```

## Others

### Write code in CommonJS modules

If you want to use CommonJS modules, `require` the code from `@puppeteer/replay/cjs`. For example,

```js
const { stringify } = require('@puppeteer/replay/cjs');
```

### Create a Chrome extension for Recorder

> You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extending DevTools.

Reviewer 2: ```suggestion
You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extend DevTools.
```

This repository contains an [example extension](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension). Once installed, the Recorder will have a new export option **Export as a Custom JSON script** in the [export dropdown](https://developer.chrome.com/docs/devtools/recorder/#export-flows).

> To load the example into Chrome DevTools. Follow these steps:-

Reviewer 2: ```suggestion
To load the example into Chrome DevTools. Follow these steps:
```

1. Download the [chrome-extension](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension) folder.
2. [Load the folder as unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/#unpacked) in Chrome. 
3. [Open a recording](https://developer.chrome.com/docs/devtools/recorder/#record) in the Recorder.

> 4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). A new **Export as a Custom JSON script** menu is added.

Reviewer 2: `A new **Export as a Custom JSON script** menu is added` sentence feels a bit out of the flow. Maybe we can say `Now you can see a new **Export as a Custom JSON script** action in the export menu`. WDYT?

> 4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). A new **Export as a Custom JSON script** menu is added.

Author: ```suggestion
4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). Now you can see a new **Export as a Custom JSON script** action in the export menu.
```

> 4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). Now you can see a new **Export as a Custom JSON script** action in the export menu.

Author: ```suggestion
4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). Now you can see a new **Export as a Custom JSON script** option in the export menu.
```

Click and watch the video demo below:

> [![Watch the video demo](https://user-images.githubusercontent.com/5917927/172872574-15ad8bea-142a-4972-bf1d-bf1379a955ba.png)](https://youtu.be/TCxIfbxgypQ)

Reviewer 2: nit: as far as I understand, this `[]` part is added as an `alt` tag to the image. So, we can try to be descriptive there instead of action oriented. We can say something like `Demo video that shows how to extend export options in Recorder panel by adding a Chrome extension`. WDYT?

> [![Watch the video demo](https://user-images.githubusercontent.com/5917927/172872574-15ad8bea-142a-4972-bf1d-bf1379a955ba.png)](https://youtu.be/TCxIfbxgypQ)

Author: ```suggestion
[![Demo video that shows how to extend export options in Recorder panel by adding a Chrome extension](https://user-images.githubusercontent.com/5917927/172872574-15ad8bea-142a-4972-bf1d-bf1379a955ba.png)](https://youtu.be/TCxIfbxgypQ)
```
